doculab/docs/api-duplicate-prevention.textile

```diff
@@ -0,0 +1,59 @@
+When making API requests, it is possible to receive an error even though your request actually completed successfully.
+
+For example, if you receive a @502 Bad Gateway@ or @504 Gateway Timeout@ error, you can't be sure whether your request was received by Chargify, or not.
```

Comment:
A 502 is almost surely returned when the request could not be processed at all, so the duplicate protection is actually not necessary/appropriate in that case.

It'd be much better to target this explanation towards when a **NETWORK** based timeout occurs.  That is, when the HTTP request was submitted but **NO** response or response code was received at all.  These situations are when it is most important to use duplicate prevention